content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md

```diff
@@ -188,7 +188,7 @@ kubeadm init [flags]
       <td colspan="2">--service-dns-domain string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "cluster.local"</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Use alternative domain for services, e.g. "myorg.internal".</td>
```

Comment:
i don't think we should touch `/generated` files with the non-latin changes.
these contents currently come from the kubernetes/kubernetes repository and latin abbreviations are considered fine there.

Reply:
I think it's OK to file an issue in [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) about making a change there to avoid Latin abbreviations.

I'd say though that it's only worth doing by people who are willing to open a PR to fix it and to follow those PR(s) through to a merge.

Reply:
the non-latin policy might not be well received by everyone.
i don't have a strong preference.